src/modules/kvs/commit.c

```diff
@@ -253,10 +254,13 @@ static int commit_unroll (commit_t *c, int current_epoch, json_t *dir)
                     goto done;
```

Comment:
oops, this is all the way back from my commit API refactoring.  Will add a cleanup.